release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -82,6 +81,13 @@ Deploying a three-node cluster is supported on Nutanix as of {product-title} {pr
 
 For more information, see xref:../installing/installing_nutanix/installing-nutanix-three-node.adoc#installing-nutanix-three-node[Installing a three-node cluster on Nutanix].
 
+[id="ocp-4-14-dc-build-apis-disabled"]
+==== DeploymentConfig and Build APIs can now be disabled
```

Comment:
```suggestion
==== Disabling DeploymentConfig and Build APIs
```

Reply:
@agantony Since this is a release note, and not about how to disable the APIs, the title looks fine to me.